packs/data/equipment.db/immovable-tripod.json

```diff
@@ -35,7 +35,13 @@
             }
         },
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "domain": "attack",
+                "key": "RollOption",
+                "option": "self:ignore-kickback-penalty"
+            }
```

Comment:
Suppress the penalty instead (selector of "strike-attack-roll" and slug of "kickback")

Reply:
```
{
    "key": "AdjustModifier",
    "suppress": true,
    "selector": "strike-attack-roll",
    "slug": "kickback"
}
```

Doesn't work.